listener/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "listener"
+version = "0.1.0"
+authors = ["Alexander Herrmann <alex@gnosis.io>", "Ben Smith <ben@gnosis.io>", "Felix Leupold <felix@gnosis.io>"]
+edition = "2018"
+
+[[bin]]
+name = "listener"
+path = "src/main.rs"
+
+[dependencies]
```

Comment:
I see there are a lot of pinned dependencies related to the graph. Do we expect to be permanently pinned to this version (if it suffices)?

Reply:
If appropriate/necessary we can upgrade them all at the same time. Given that they point at a tagged version and not a random commit I'm pretty confident this version is somewhat stable.